src/Reflex/Dom/Contrib/Widgets/ButtonGroup.hs

```diff
@@ -167,15 +167,15 @@ radioGroup dynName dynEntryList cfg = do
      <> bool mempty ("checked" =: "checked") chkd
     handleOne _ dynV dynChecked = do
 
-      el "tr" $ do
+      el "tr" $ el "td" $ el "label" $  do
```

Comment:
Good point.  We're already going to do a major version bump for the next release, so now is a good time to sneak this in.  :)